src/testHelpers/createMockConfig.js

```diff
@@ -25,6 +25,16 @@ const createMockConfig = ({ clientSide } = {}) => {
       databaseURL: 'https://my-example-app.firebaseio.com',
       projectId: 'my-example-app-id',
     },
+    redirectConfig: {
+      authenticatedUser: {
```

Comment:
I like the names you chose here, but wondering what the lib author will have to say since it seems like with other config he refers to them as authPage and appPage

Reply:
The logic for redirects here is a little weird. The consumers of the lib have to configure the redirects in somewhat a restricted framework. For example, if the user is unauthorized, the consumer needs to add the proper whenUnauthed / whenAuthed config with the right authPage or appPage. It is not really clear that those specific configurations cause the redirect. So the intention here was to make it more clear, the configs here being the global configs and the `redirectConfig` as an argument to the HOC allowing an override.

Reply:
I actually think I see a deeper point here. This proposal is not really consistent with the terms in the library. I'll go back and think about them a bit. There is `whenAuthed` and `whenUnauthed` everywhere. 😬